app/templates/components/public/ticket-list.hbs

```diff
@@ -61,8 +61,8 @@
 </div>
 <div class="ui segment">
   <div class="ui grid">
-    <div class="ui row grid">
-      <div class="ui padding less fourteen wide column right aligned floated payment icons">
+    <div class="ui row {{if device.isMobile 'stackable'}} grid">
```

Comment:
Stackable can always be there. it would come into effect only on smaller screens.